src/building/how-to-build-and-run.md

```diff
@@ -82,6 +79,33 @@ incremental = true
 verbose-tests = true
 ```
 
+Moreover, there are a couple of other configs to consider, depending on whether
+you are planning to work on something LLVM-related.
+
+If you are planning to work on LLVM-related things, you may want to enable
+assertions for improved debugging. This comes at a cost in compile time:
```

Comment:
It would be great to know how _much_ of a cost :/ but we've already discussed that we don't know. Other than that this looks great.